docs/instance_groups.md

```diff
@@ -239,6 +239,25 @@ https://docs.aws.amazon.com/autoscaling/ec2/APIReference/API_InstancesDistributi
 Used only when the Spot allocation strategy is lowest-price.
 The number of Spot Instance pools across which to allocate your Spot Instances. The Spot pools are determined from the different instance types in the Overrides array of LaunchTemplate. Default if not set is 2.
 
+### instanceRequirements
+
+{{ kops_feature_table(kops_added_default='1.24') }}
+
+Instead of configuring specific machine types, the InstanceGroup can be configured to use all machine types that satisfies a given set of requirements.
```

Comment:
```suggestion
Instead of configuring specific machine types, the InstanceGroup can be configured to use all machine types that satisfy a given set of requirements.
```